libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/PrimitivesEx.kt

```diff
@@ -0,0 +1,13 @@
+package com.mapbox.navigation.base.internal.extensions
+
+fun Double?.equals(other: Double?): Boolean {
```

Comment:
This conflicts with the built-in `equals` function. 
In case of a non-null receiver a built-in function will be used instead.